mods/ts/maps/sunstroke/sunstroke.lua

```diff
@@ -1,3 +1,11 @@
+--[[
+   Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
 Tick = function()
```

Comment:
Does it have any effect if we put a license header on something which is not copyrightable?